src/core/Slider/DaySlider.tsx

```diff
@@ -71,6 +83,9 @@ class DaySlider extends React.Component<Props, State> {
   static defaultProps = {
     variableWidth: false,
     swipeToSlide: true,
+    afterSlideChange: () => {},
+    showLeftArrow: true,
+    showRightArrow: true,
   };
 
   static Slide = (props: SlideProps) => (
```

Comment:
No se masa be si es un problema de este component o de algun altre... pero en algunes resolucions els dies del slider no aplegen al final... disant molt de espai buit

![144](https://user-images.githubusercontent.com/4190084/48942717-01535d80-ef21-11e8-9388-4eb1d4199db6.png)

Resolucions en les que me pasa (soles width):
* 1024px
* 768px
* 425px
* 320px (en esta es un poc menos)

La resta esta perfecte, en diseny i en funcionalitat